### PR TITLE
WEBSITE-344 Update the license info in README to mention CC and ASL

### PR DESCRIPTION
**Pull request by Sanne**
I'm trying to differentiate between the blog content and the code which actually builds the website. I hope it's clear enough?

https://hibernate.atlassian.net/browse/WEBSITE-344

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -321,7 +321,10 @@ Panic! Then completely regenerate the site via:
 
 == License
 
-The content of this repository is released under the link:http://www.apache.org/licenses/LICENSE-2.0.txt[ASL 2.0].
+The articles, blog posts and other content of this repository is released under the link:http://creativecommons.org/licenses/by-sa/3.0/[Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)].
+
+Sample code available on this content is released under the link:http://www.apache.org/licenses/LICENSE-2.0.html[Apache Software License 2.0].
```
Comment (hferentschik) [inline README.adoc]: "on this content" - do you mean "on this site"?

File: README.adoc [hunk 2]
```diff
@@ -321,7 +321,10 @@ Panic! Then completely regenerate the site via:
 
 == License
 
-The content of this repository is released under the link:http://www.apache.org/licenses/LICENSE-2.0.txt[ASL 2.0].
+The articles, blog posts and other content of this repository is released under the link:http://creativecommons.org/licenses/by-sa/3.0/[Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)].
```
Comment (hferentschik) [inline README.adoc]: is == are

Comment (hferentschik) [inline README.adoc]: Either drop the 'the' and just "released under Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)" or keep the 'the' but then it is most likely better with "released under the Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0) license."

Comment (hferentschik) [inline README.adoc]: articles, blog posts and other content - what the difference to you between articles and blog posts. Are this not all just blog posts?

File: README.adoc (hunk 1 above)
Comment (Sanne) [inline README.adoc]: No I'm referring to the content mentioned in the previous paragraph. This README is tricky, as it refers to both the generated and published site, and to the source code repository. Not sure how to make the clear distinction?

File: README.adoc (hunk 2 above)
Comment (Sanne) [inline README.adoc]: > articles, blog posts and other content - what the difference to you between articles and blog posts. Are this not all just blog posts?

I guess. Not sure if all what we published falls under the category "blog", not sure if that's a legal term either..

**Comment by Sanne** [comment]
Pushed a new version. Thanks for the corrections!

**Comment by emmanuelbernard** [comment]
Applied.

**Pull request closed without merging**